CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
 # CHANGELOG
+## 3.4.0
+#### New Functionality
+
+- Can now talk to MarkLogic using OAuth token (MarkLogic Server version 11.2.0 or higher needed).
+- Can now submit optic update plans to /v1/rows/update (MarkLogic Server version 11.2.0 or higher needed).
```

Comment:
Capitalize "Optic Update". And given that we don't have any other documentation for this, I think a brief mention of how this is accomplished is worth including. The user doesn't actually need to know about the "v1/rows/update" endpoint. 

Instead, how about something like: "Optic Update plans, including DSL queries that perform updates, now require `update: true` to be included in the options sent to the `rows.execute()` function."